single_lwr_example/single_lwr_moveit/launch/moveit_planning_execution.launch

```diff
@@ -0,0 +1,78 @@
+<launch>
+  <!-- The planning and execution components of MoveIt! configured to run -->
+  <!-- using the ROS-Industrial interface. -->
+ 
+  <!-- Non-standard joint names:
+       - Create a file [robot_moveit_config]/config/joint_names.yaml
+           controller_joint_names: [joint_1, joint_2, ... joint_N] 
+       - Update with joint names for your robot (in order expected by rbt controller)
+       - and uncomment the following line: -->
+  <!-- <rosparam command="load" file="$(find [robot_moveit_config])/config/joint_names.yaml"/> -->
+ 
+  <!-- the "sim" argument controls whether we connect to a Simulated or Real robot -->
+  <!--  - if sim=false, a robot_ip argument is required -->
+  <arg name="sim" default="true" doc="If true, the robot will be simulated in Gazebo" />
```

Comment:
By `sim`, they mean to use the `industrial_robot_simulator`, is it ok to change to gazebo?

I think this can be one difference that justify having this file.